pkg/statedb/webhook.go

```diff
@@ -0,0 +1,100 @@
+package statedb
+
+import (
+	"fmt"
+	"time"
+
+	"gorm.io/datatypes"
+)
+
+type Webhook struct {
+	ID      uint   `gorm:"column:id;primarykey"`
```

Comment:
We use string UUIDs for primary keys elsewhere in the codebase for non-atproto stuff, let's keep that here for consistency
```suggestion
	ID      string   `gorm:"column:id;primarykey"`
```